config/jobs/ci-infra/build-ci-infra-images.yaml

```
@@ -0,0 +1,49 @@
+postsubmits:
+  gardener/ci-infra:
+  - name: post-build-ci-infra-images
```

Let's keep the pattern `{pull,post,ci}-<repo>-<job-name>`
```suggestion
  - name: post-ci-infra-build-images
```